samples/csharp/MicrosoftEduGraphSamples/Workflows/SubmissionWorkflow.cs

```diff
@@ -146,5 +146,67 @@ public async Task ReassignWorkflow()
                 Console.WriteLine($"ReassignWorkflow: {ex.ToString()}");
             }
         }
+
+        /// <summary>
+        /// Workflow to create a batch request and get the responses
```

Comment:
I would expect here and in Readme a more detailed context on the scenario.
i.e. to convey something like this - Since as a teacher you can't expand submissions, you will have to request every assignment for submissions on it. This SDK sample is to illustrate on how you can do this in a single batch request.

Reply:
correct, even that dsescription had to be included in the readme file, added now